### ISSUE TITLE
Add leap.nvim - like extension

### ISSUE THREAD

**Issue by donalffons** [zed-industries/extensions#939]
### Check for existing issues

- [X] Completed

### Misc notes

I love using [leap.nvim](https://github.com/ggandor/leap.nvim) in neovim. Having this available in zed's vim mode would be  wonderful :-).

**Comment by naheller** [comment]
Yep similar to how Zed uses bits of vim-surround, a little leaping would be excellent!
